WMF Framework/NavigationBar.swift

```diff
@@ -8,7 +8,7 @@
 @objc(WMFNavigationBar)
 public class NavigationBar: SetupView, FakeProgressReceiving, FakeProgressDelegate {
     fileprivate let statusBarUnderlay: UIView =  UIView()
-    fileprivate let titleBar: UIToolbar = UIToolbar()
+    fileprivate let titleBar: UIToolbar = UIToolbar(frame: .infinite) // HAX: `.infinite` is workaround for iOS 13 constraint breakage
```

Comment:
Setting this to a value that isn't valid for view frames like `CGRect.infinite` could lead to other unexpected behavior down the line. A fix that'd be more in line with expected behavior for views would be to have `NavigationBar` set all the subviews to the initial bounds in `init(frame:` and provide an arbitrary, valid default frame for `NavigationBar` like `CGRect(x: 0, y: 0, width: 100, height: 44)`

Reply:
Good point. `CGRect.zero` was causing problems so it's probably safer to assume `CGRect.infinite` could have issues too.